hasura/metadata/tables.yaml

```diff
@@ -435,7 +425,6 @@
       - availability_hours
       - color_mask
       - player_type_id
-      - profile_layout
```

Comment:
@alalonde why are we removing this ?

Reply:
Hmm, that snuck in there again. There should be no changes in this file. My local hasura metadata is messed up.